src/Component.php

```diff
@@ -50,7 +51,7 @@ protected function run(): void
                 break;
             case 'child-jobs':
                 $timeout = $config->getTimeout();
-                $queueClient = new Client($this->getLogger(), $config->getQueueApiUrl(), $config->getToken());
```

Comment:
Tohle je trochu škoda ne? Ten logger zmizel z konstruktoru, ale dá se přidat přes options.

https://github.com/keboola/job-queue-api-php-client/blob/02c25cee87fc8a80b5b2e07c2618bf48d646a1d0/src/Client.php#L136

Reply:
ono je to sice trošku něco jinýho, ale nezaškodí